__main__.py

```diff
@@ -1,11 +1,11 @@
 import app
+import database
 
-
-if __name__ == "__main__":
+if _name_ == "_main_":
```

Comment:
`__name__` nunca va a poder ser `_main_`

[Pequeño articulo](https://www.freecodecamp.org/news/whats-in-a-python-s-name-506262fe61e8/#:~:text=The%20__name__%20variable%20(two%20underscores%20before%20and%20after,a%20module%20in%20another%20script.)